src/pages/index.js

```diff
@@ -43,45 +23,198 @@ const enableValidation = (config) => {
   const formList = Array.from(document.querySelectorAll(config.formSelector));
   formList.forEach((formElement) => {
     const validator = new FormValidator(config, formElement);
-    // here you get the name of the form
+    // get the name of the form
     const formName = formElement.getAttribute("name");
 
-    // here you store a validator by the `name` of the form
+    // store a validator by the `name` of the form
     formValidators[formName] = validator;
     validator.enableValidation();
   });
 };
 
 enableValidation(config);
 
```

Comment:
Needs correction: I don't see loading on the form buttons while the request is in progress